.github/workflows/release.yml

```diff
@@ -33,5 +35,50 @@ jobs:
           name: nugets
           path: nugets/*
           retention-days: 1
-      - name: Deploy to MyGet
-        run: dotnet nuget push nugets/Particular.EndpointThroughputCounter.${{env.MinVerVersion}}.nupkg --source https://www.myget.org/F/particular/api/v2/package --api-key ${{ secrets.MYGET_PUBLISH_API_KEY }}
+     #- name: Deploy to MyGet
+       #run: dotnet nuget push nugets/Particular.EndpointThroughputCounter.${{env.MinVerVersion}}.nupkg --source https://www.myget.org/F/particular/api/v2/package --api-key ${{ secrets.MYGET_PUBLISH_API_KEY }}       
+      - name: Build SelfContained
+        run: |
+          dotnet publish src/tool/Particular.EndpointThroughputCounter.csproj -r win-x64 `
```

Comment:
You got:

> MSBUILD : error MSB1009: Project file does not exist.

I think because Linux is case sensitive and the directory should be `Tool`